src/js/state.js

```diff
@@ -41,6 +41,10 @@ export default async function () {
   } else {
     ciphers = ciphers;
   }
+  if (protocols.includes('TLSv1.1') && configs[server].usesOpenssl !== false && minver('3.0.0', form['openssl'].value)) {
+    // to allow "old" TLSv1-1.1 with OpenSSL 3.x we add SECLEVEL=0 via cipher string
+    if (!ciphers.includes('@SECLEVEL=0') && ciphers.length) ciphers.unshift('@SECLEVEL=0');
+  }
 
```

Comment:
And shift when switching states:

```suggestion
  }
  if (protocols.includes('TLSv1.1') && configs[server].usesOpenssl !== false && minver('3.0.0', form['openssl'].value)) {
    // to allow "old" TLSv1-1.1 with OpenSSL 3.x we add SECLEVEL=0 via cipher string
    if (!ciphers.includes('@SECLEVEL=0') && ciphers.length) ciphers.unshift('@SECLEVEL=0');
  } else {
    if (ciphers[0] === '@SECLEVEL=0') ciphers.shift();
  }

```

Reply:
This should be done unconditionally before the logic which might add it, and also needs to check ciphers.length, since ciphers.length might be 0, e.g. with the current config for Modern with TLSv1.3.

Reply:
- No need as it's either–or, no other case so if it's there it's tested and not added again; no need to remove beforehand, and only need to remove if it's there and shouldn't (not touching it otherwise on other state changes)
- I originally had `.at(0)` to not have worry about indexes/prototype etc.; but that doesn't have good browser support — need to check if that's something `babel` takes care of for us.
- Not sure if happy trailing `ciphers?.[0]` evaluates better.
- Basically no matter what type the var is, accessing the index even for empty array or a number, string or bool ends up falsy (no error is raised if that's what you're concerned about), the shift only happens when it's safe to do.

Reply:
FWIW instead of shifting an already manipulated cipher list, it'd be better to find why it's not being populated fresh on every state change from `ssc.ciphers.openssl` (that it looks like it should, it just keeps referencing the already changed version from the past state(?))